dynatrace-oneagent-operator/values.yaml

```diff
@@ -44,6 +44,8 @@ oneagent:
   proxy: ""
   trustedCAs: ""
   networkZone: ""
+  useUnprivilegedMode: false
+  useImmutableImage: false
```

Comment:
Again, agentVersion, and customPullSecret?

Reply:
Added